src/t2iapi/alert/alert_requests.proto

```diff
@@ -34,6 +34,20 @@ message SetAlertSignalPresenceRequest {
   AlertSignalPresence presence = 2;  // alert signal presence to set
 }
 
+/*
+Request to set the pm:AlertSignalState/@Presence either for all provided alert signals or only for a subset of the
+provided alert signals to the provided @Presence. If more than one @Presence value is provided, all manipulated alert
+signals shall have their @Presence randomly set to all values provided by the parameter "presence". The minimal
```

Comment:
It cannot be set to all values, only one. Is it supposed to switch randomly, or choose one randomly and keep it?